vendor/assets/javascripts/angular_progress/app.js.coffee

```diff
@@ -1,4 +1,5 @@
 angular.module('AngularProgress', [])
 angular.module('AngularProgress').config ['$httpProvider', ($httpProvider) ->
-  $httpProvider.interceptors.push('httpProgressInterceptor')
```

Comment:
Can we make this
`$httpProvider.interceptors?.push('httpProgressInterceptor')` please?